### PR TITLE
1104 export csv observed data

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.CLI.Core/Services/QualificationRunner.cs [hunk 1]
```diff
@@ -160,25 +163,34 @@ private IEnumerable<PlotMapping> retrieveChartDefinitionsForSimulation(Simulatio
...
+         return Task.WhenAll(project.AllObservedData.Select(x => exportObservedData(x, configuration, project)));
+      }
+
+      private async Task<ObservedDataMapping> exportObservedData(DataRepository observedData, QualifcationConfiguration configuration, PKSimProject project)
+      {
+         var observedDataOutputFolder = configuration.ObservedDataFolder;
+         var removeIllegalCharactersFrom = FileHelper.RemoveIllegalCharactersFrom(observedData.Name);
+         var csvFullPath = Path.Combine(observedDataOutputFolder, $"{removeIllegalCharactersFrom}{Constants.Filter.CSV_EXTENSION}");
+         var xlsFullPath = Path.Combine(observedDataOutputFolder, $"{removeIllegalCharactersFrom}{Constants.Filter.XLSX_EXTENSION}");
+         _logger.AddDebug($"Observed data '{observedData.Name}' exported to '{csvFullPath}'", project.Name);
```
Comment (msevestre) [inline src/PKSim.CLI.Core/Services/QualificationRunner.cs]: export to csv and excel

File: src/PKSim.CLI.Core/Services/QualificationRunner.cs [hunk 2]
```diff
@@ -160,25 +163,34 @@ private IEnumerable<PlotMapping> retrieveChartDefinitionsForSimulation(Simulatio
...
-            _logger.AddDebug($"Observed data '{obs.Name}' exported to '{fileFullPath}'", project.Name);
-            _dataRepositoryTask.ExportToExcel(obs, fileFullPath, launchExcel: false);
-            return new ObservedDataMapping
-            {
-               Id = obs.Name,
-               Path = relativePath(fileFullPath, configuration.ReportConfigurationFile)
-            };
-         }).ToArray();
+            Id = observedData.Name,
+            Path = relativePath(csvFullPath, configuration.ReportConfigurationFile)
```
Comment (msevestre) [inline src/PKSim.CLI.Core/Services/QualificationRunner.cs]: Mapping will use csv however!

**Comment by Yuri05** [comment]
1 test failed

**Comment by msevestre** [comment]
This test has nothing to do with the changes. Weird.
Going to trigger again and see

**Pull request merged**